### ISSUE TITLE
Passive Voice? I think not

### ISSUE THREAD

**Issue by galbaras** [Yoast/YoastSEO.js#2141]
* [x] I've read and understood the [contribution guidelines](https://github.com/Yoast/wordpress-seo/blob/trunk/.github/CONTRIBUTING.md).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened.

The plugin flags the following sentences as Passive Voice, but I disagree:

1. Years later, I realized that if I really wanted to be on that Olympic team, I would have found a way.

2. Some girls trained for 5-6 years before they got into the competitive team and had to hang in there long enough to move up.

### Technical info

* If relevant, which editor is affected (or editors): 
- [ ] Classic Editor
- [ ] Gutenberg
- [x] Classic Editor plugin

* Which browser is affected (or browsers): 
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other 

#### Used versions
* WordPress version: 5.0.3
* Yoast SEO version: 9.5
* Classic Editor plugin version: 1.4
* Tested with theme: Genesis + custom child

**Comment by galbaras** [comment]
Come to think of it, it would be really good to have a way to un-flag a sentence and/or send it to the plugin's team from within the analysis.

Having red scores is stressful :-)

**Comment by monbauza** [comment]
Another false positive of sentence being flagged as passive: 
> The wet part was not wholly unexpected.

**Comment by monbauza** [comment]
Please inform the customer of conversation # 605408 when this conversation has been closed.

**Comment by monbauza** [comment]
The above customer doesn't think these sentences should be flagged as passive:
> Import-Export Code (IEC) registration is needed for organisations. It is not required for individuals.

**Comment by monbauza** [comment]
[This user](https://twitter.com/MiraVioIet/status/1319023267957624834) on Twitter doesn't think this sentence is in passive voice:
> Rules are meant to be broken.

**Comment by galbaras** [comment]
@monbauza all of your examples are passive. Active equivalents would be:

I half expected to see a wet part

Organisations need to register their Import-Export Code (IEC), but individuals don't

We should break rules

**Comment by galbaras** [comment]
> @galbaras You are totally correct those sentences are not passive. Unfortunately, our analysis is not 100% correct, and some false positives can occur. We'll see what we can do based on your examples. I'm going to move this issue to the yoastseo.js repository, because that's where our analysis takes place.

@IreneStr The most annoying issue is with feelings, like "I felt depressed" or "I got upset". Maybe you can skip known feeling words?

Wither way, this issue has been open for more than 1.5 years. Has there been ANY progress?

**Comment by rumejan** [comment]
Please inform the customer of conversation # 661962 when this conversation has been closed.

**Comment by IreneStr** [comment]
Hi @galbaras,
Thank you for your feedback. We understand the frustration of having non-passive sentences marked as passive. Based on your suggestion above, we’ve added ‘depressed’ and ‘upset’ to our exception list. We also added 'annoyed' and 'disappointed'. If you have more of those words we can add to this list, please let us know. Adding words to the exceptions list is a quick fix for us. 

The reason this issue has been open for such a long time is that we have decided to focus on expanding our readability features to other languages, instead of decreasing false positives for existing implementations. Unfortunately, we don’t have enough developers and linguists to do both. (We’ll always be able to add words to the exception words list if someone provides them to us, but not to do research for it ourselves).

Please note that what our assessments indicate is not (and will never be) 100% consistent with what Google or your readers will think of your posts. If you yourself know that sentences are incorrectly marked as passive, you can keep those sentences without a doubt. They are not passive, so they won’t make your text harder to read for Google or your readers.

**Comment by galbaras** [comment]
Thank you, @IreneStr .

It could be good to skip anything following a word that indicates feeling, e.g. "I **felt** satisfied".

Also, I've found [this list of feeling words](http://www.psychpage.com/learning/library/assess/feelings.html). If you search the page for "ed", you'll find MANY words to exclude, like "elated", "pleased" and "thrilled".

**Comment by IreneStr** [comment]
@galbaras We already check if there is a passive auxiliary present in the sentence. So "I felt satisfied" should never be marked as passive (but "I was satisfied" might). Did you see "I felt satisfied" being marked?

It's risky to remove all words from your list of feeling words ending in -ed, as that would cause many false negatives as well. For example, the first -ed word there is 'amazed'. A sentence like 'I was amazed by the magician's tricks' is passive, and my gut feeling says that sentences with 'amazed' + 'by' (which are passive) are more frequent than sentences with 'amazed' without 'by' (e.g 'I was amazed', which is not passive). And exactly for that kind of evaluation/investigation we don't have enough linguists 😬

**Comment by galbaras** [comment]
> @galbaras We already check if there is a passive auxiliary present in the sentence. So "I felt satisfied" should never be marked as passive (but "I was satisfied" might). Did you see "I felt satisfied" being marked?

That's great. I don't remember specifically, sorry.

> It's risky to remove all words from your list of feeling words ending in -ed, as that would cause many false negatives as well. For example, the first -ed word there is 'amazed'. A sentence like 'I was amazed by the magician's tricks' is passive, and my gut feeling says that sentences with 'amazed' + 'by' (which are passive) are more frequent than sentences with 'amazed' without 'by' (e.g 'I was amazed', which is not passive). And exactly for that kind of evaluation/investigation we don't have enough linguists 😬

In this case, having "by" after the supposedly-passive word can be used in the analysis.

Either way, taking the unambiguous feeling words from the list should already make things better.

Just as an idea, BTW, why not provide a way to dispute readability analysis in the readability box? You'll only need it for a while, until the dust settles.

Basically, users can send you some sample text and a description of why they think it should be analysed differently, and clicking a button will send that feedback to be collected somewhere, and then reviewed.
